File: Source/Aladdin.ASN1.ISO.Java/src/aladdin/asn1/iso/pkix/Name.java
package aladdin.asn1.iso.pkix;
import aladdin.asn1.*; 

// Name ::= CHOICE { rdnSequence RelativeDistinguishedNames }

public final class Name extends Choice
{
	// информация о структуре
	private static final ObjectInfo[] info = new ObjectInfo[] { 

		new ObjectInfo(new ObjectCreator(RelativeDistinguishedNames.class).factory(), Cast.N), 
	}; 
	// конструктор
	public Name() { super(info); } 
}

File: Source/Aladdin.ASN1.ISO/ASN1/ISO/PKIX/Name.cs
using System;

// Name ::= CHOICE { rdnSequence RelativeDistinguishedNames }

namespace Aladdin.ASN1.ISO.PKIX
{
	public class Name : Choice
	{
		// информация о структуре
		private static readonly ObjectInfo[] info = new ObjectInfo[] { 

			new ObjectInfo(new ObjectCreator<RelativeDistinguishedNames>().Factory(), Cast.N), 
		}; 
		// конструктор
		public Name() : base(info) {} 
	}
}
